whois-commons/src/main/java/net/ripe/db/whois/common/DatabaseDummifierJmx.java

```diff
@@ -84,6 +86,10 @@ public String call() {
                 final SimpleDataSourceFactory simpleDataSourceFactory = new SimpleDataSourceFactory("org.mariadb.jdbc.Driver");
                 final DataSource dataSource = simpleDataSourceFactory.createDataSource(jdbcUrl, user, pass);
                 jdbcTemplate = new JdbcTemplate(dataSource);
+
+                final DataSource internalSource = simpleDataSourceFactory.createDataSource("jdbc:mariadb://localhost/INTERNALS_RIPE", user, pass);
```

Comment:
Is host always `localhost`? Or do we need to adjust the JDBC URL ?

Reply:
yes, unless that we decide to run this script in Sussex for example and to use the Internals of Dexter for example (in that case it would be dexter.ripe.net instead of localhost). But II don't know why we would do that. Even that we will run this script always in master database (we are updating the database) and master will always have that database.

I can request the internal database as parameter as we do with whois_update database.

Reply:
No it's fine! Thanks for confirming